jupyterhub/handlers/base.py

```diff
@@ -621,8 +621,8 @@ def set_login_cookie(self, user):
     def authenticate(self, data):
         return maybe_future(self.authenticator.get_authenticated_user(self, data))
 
-    def load_user_groups(self, auth_state):
-        return maybe_future(self.authenticator.load_user_groups(auth_state))
+    def load_user_groups(self, auth_state, user):
+        return maybe_future(self.authenticator.load_user_groups(user, auth_state))
```

Comment:
Let's be consistent with the argument order - this call has `user, auth_state`, whereas the definition has `auth_state, user`. I think user first makes the most sense.

Let's also be consistent about variable names - `auth_state` means something specific (the persisted value from `await user.get_auth_state()`, whereas this is being called with the return value of `authenticate()`, which is a dict that may _contain_ auth state, but is not auth_state itself. We've called this dict `auth_info` elsewhere `auth_info (dict): the return value of Authenticator.authenticate() or Authenticator.refresh_user()`.